astropy/io/fits/column.py

```diff
@@ -1095,7 +1095,7 @@ def _convert_to_valid_data_type(self, array):
             elif 'L' in format:
                 # boolean needs to be scaled back to storage values ('T', 'F')
                 if array.dtype == np.dtype('bool'):
-                    return np.where(array == False, ord('F'), ord('T'))
+                    return np.where(array == np.bool(False), ord('F'), ord('T'))
```

Comment:
I think here the comparison is correct as written, which would suggest a pylint disable ... ...  

Or, how about `array == np.False_`? Slightly more readable than `np.bool(False)